.github/workflows/test.yml

```diff
@@ -28,12 +28,12 @@ jobs:
               with:
                 repository: boschresearch/bt_tools
                 ref: feature/fsm_conversion
-                path: colcon_ws/src/bt_tools
+                path: /colcon_ws/src/bt_tools
```

Comment:
It seems GH actions do not like global paths :(
I made it relative again and made sure we test only the pkgs we need